packages/appkit/package.json

```diff
@@ -100,33 +99,20 @@
     "@reown/appkit-wallet": "workspace:*",
     "@walletconnect/types": "2.16.1",
     "@walletconnect/utils": "2.16.1",
+    "@walletconnect/universal-provider": "2.16.1",
     "valtio": "1.11.2"
   },
-  "optionalDependencies": {
-    "borsh": "0.7.0",
-    "bs58": "5.0.0"
-  },
   "devDependencies": {
     "@types/react": "18.2.0",
     "@types/react-dom": "18.2.0",
     "@vitest/coverage-v8": "2.0.5",
     "@vue/runtime-core": "3.4.3",
-    "@wagmi/connectors": "5.1.9",
-    "@wagmi/core": "2.13.4",
-    "@wallet-standard/app": "1.0.1",
-    "@wallet-standard/base": "1.0.1",
-    "@wallet-standard/features": "1.0.3",
-    "@wallet-standard/wallet": "1.0.1",
```

Comment:
why remove all of these ?

Reply:
We do need them in Solana adapter. Do we need adapter specific packages in `/appkit` package?